packages/strapi-plugin-content-type-builder/utils/helpers.js

```diff
@@ -61,10 +61,15 @@ const reorderList = (manager, list) => {
   return List(flattenDeep(reordered));
 };
 
+const escapeNewlines = (content, placeholder = '\n') => {
```

Comment:
Do we need export this function in a helper ?

Reply:
Not really. When reading the code I thought it was the right place for this.